site/vocs.config.tsx

```diff
@@ -20,7 +20,7 @@ export default defineConfig({
   },
   iconUrl: "/kit-icon.svg",
   rootDir: "./",
-  ogImageUrl: "/images/og-image.jpg",
+  ogImageUrl: "https://accountkit.alchemy.com/images/og-image.jpg",
```

Comment:
From the vocs docs it looks like this needs to be the full url to work.  I checked in the Vercel Open Graph debugger and it looks like it is working now.